tests/test_writers.py

```diff
@@ -84,3 +84,18 @@ def test_write_sssom_owl(self):
         # FIXME this test doesn't test anything
         # TODO implement "read_owl" function
         self.assertEqual(1, 1)
+
+    def test_write_sssom_ontoportal_json(self):
+        """Test writing as ontoportal JSON."""
+        path = os.path.join(test_out_dir, "test_write_sssom_ontoportal_json.json")
```

Comment:
This file seems to be present locally and not version controlled. If this test is run on another system, it may fail (?) unless if its generated by some other code in the test. Please correct me if I'm wrong. Version controlled test data could be put in `tests/data/` directory.